src/modules/displayScore.js

```diff
@@ -0,0 +1,16 @@
+import getScore from './getScore.js';
+
+const scoreContainer = document.querySelector('.scores-container');
+
+const printScore = async () => {
+  const result = await getScore();
+  const p = document.createElement('p');
+
+  result.forEach((game) => {
+    p.className = 'user-score';
+    p.innerHTML = `${game.user}: ${game.score}`;
+    scoreContainer.appendChild(p);
+  });
+};
+
+export default printScore;
```

Comment:
- while posting & retrieving data is working correctly, you have a bug in your display function that only display one  result and not all of the scores 

- [ ] kindly fix the issue and make sure to append the result only once and refresh all of them at once when clicking the button, here is an example of the bug : 

![BugLeadboard](https://user-images.githubusercontent.com/89282221/210774354-6c085d62-29b4-42ec-93ed-d5dab80a3f80.gif)

Reply:
Hey @cherelemma I hope this issue is not fixed yet 😨